resources_pages/install_ds_stack_mac.md

```diff
@@ -232,39 +242,35 @@ To enable us to access the most up to date version of the Python packages we are
 we will add the more up to date  channel,
 To add the conda-forge channel by typing the following in the terminal:
 
-
-```
+```bash
 conda config --add channels conda-forge
-
 ```
 
 To install packages individually, we can now use the following command: `conda install <package-name>`. Let's install the key packages needed for the start of our program:
 
-```
+```bash
 conda install \
- jupyterlab=2.* \
+ jupyterlab=3.* \
  numpy=1.* \
  pandas=1.* \
  flake8=3.* \
- black=19.*
+ black=21.*
 ```
 
 `conda` will show you the packages that will be downloaded,
 and you can press enter to proceed with the installation.
 If you want to answer `yes` by default and skip this confirmation step,
 you can replace `conda install` with `conda install -y`.
 
-> Note: we will use many more packages than those listed above across the MDS program, however we will manage these using virtual environments (which you will learn about in DSCI 521: Platforms for Data Science).
+> **Note:** we will use many more packages than those listed above across the MDS program, however we will manage these using virtual environments (which you will learn about in DSCI 521: Platforms for Data Science).
 
 ## JupyterLab setup
 
 We will be using the Jupytext Python package and the JupyterLab git extension to facilitate using Jupyter notebooks with Git & GitHub. Install them via the following commands:
 
-```
-conda install nodejs=10.*
-pip install --upgrade jupyterlab-git
-conda install jupytext=1.*
-jupyter lab build
+```bash
+conda install nodejs=15.*
+conda install -c conda-forge jupyterlab jupyterlab-git jupytext
```

Comment:
Note to self to make this one line or remove the nodejs part if we go with another solution for the keyboard text shortcuts.